src/swiftcore/Info.plist

```diff
@@ -6,9 +6,13 @@
 	<string>swift core</string>
 	<key>CFBundleDisplayName</key>
 	<string>swift core</string>
+    <key>CFBundleVersion</key>
+    <string>${SWIFT_VERSION_MAJOR}.${SWIFT_VERSION_MINOR}.${SWIFT_VERSION_REVISION}</string>
+    <key>CFBundleShortVersionString</key>
+    <string>${SWIFT_VERSION_MAJOR}.${SWIFT_VERSION_MINOR}.${SWIFT_VERSION_REVISION}</string>
```

Comment:
Inconsistent indentation tabs vs spaces

Reply:
I missed to change that one. Is fixed now.